File: recovered/src/com/lumiyaviewer/lumiya/render/avatar/AnimationTiming.java
package com.lumiyaviewer.lumiya.render.avatar;

/* loaded from: classes.dex */
public class AnimationTiming {
    public float runningTime = 0.0f;
    public float inAnimationTime = 0.0f;
    public float inFactor = 0.0f;
    public float outFactor = 1.0f;

    public boolean hasStopped() {
        return this.outFactor <= 0.0f;
    }
}

File: app/src/main/java/com/lumiyaviewer/lumiya/render/avatar/AnimationTiming.kt
package com.lumiyaviewer.lumiya.render.avatar

open class AnimationTiming {
    @JvmField var runningTime = 0.0f
    @JvmField var inAnimationTime = 0.0f
    @JvmField var inFactor = 0.0f
    @JvmField var outFactor = 1.0f

    fun hasStopped(): Boolean = outFactor <= 0.0f
}
